source_gen/CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
   For questions please use https://github.com/dart-lang/build/discussions.
 - Updated the minimum package versions for a number of dependencies.
 - Require Dart 3.7.0
+- Update the GeneratorForAnnotation optimization to skip files with well known
```

Comment:
-dev is released, I guess start a block for `3.0.0`

Reply:
Updated it to `-wip` following the standard in other repos for unreleased in progress versions.